src/fidesops/task/graph_task.py

```diff
@@ -603,6 +608,13 @@ def termination_fn(
         dsk[TERMINATOR_ADDRESS] = (termination_fn, *end_nodes)
         update_mapping_from_cache(dsk, resources, start_function)
 
+        log_graph_rerun(
+            prepare_rerun_graph_analytics_event(
+                privacy_request, env, end_nodes, resources, ActionType.access
+            )
+        )
```

Comment:
I just meant not having the two function calls here, just so it would look cleaner.  have `log_graph_rerun` itself take in privacy request, env, end_nodes, etc. and internally call prepare_rerun_graph_analytics_event.  But I think I'll just leave it.